config/trustedproxy.php

```diff
@@ -25,4 +25,22 @@
      */
     'proxies' => in_array(env('TRUSTED_PROXIES', []), ['*', '**']) ?
         env('TRUSTED_PROXIES') : explode(',', env('TRUSTED_PROXIES') ?? ''),
+
+    'cloudflare' => [
+        '173.245.48.0/20',
+        '103.21.244.0/22',
+        '103.22.200.0/22',
+        '103.31.4.0/22',
+        '141.101.64.0/18',
+        '108.162.192.0/18',
+        '190.93.240.0/20',
+        '188.114.96.0/20',
+        '197.234.240.0/22',
+        '198.41.128.0/17',
+        '162.158.0.0/15',
+        '104.16.0.0/13',
+        '104.24.0.0/14',
+        '172.64.0.0/13',
+        '131.0.72.0/22',
```

Comment:
Screenshots show ipv6? But they're not listed in any of the changed files?

Reply:
You must have cherrypick the commit cause its in the second one
![image](https://github.com/user-attachments/assets/e6aae008-4e92-447a-8eab-105b3a1876cf)

Reply:
They were 100% not there when i looked lol.